### PR TITLE
MM-25083: local mode for patchBot, getBots, enableBot, disableBot and assignBot

### PR DESCRIPTION
**Pull request by ashishbhate**
#### Summary

- Local mode handler for patchBot
- Local mode handler for getBots
- Local mode handler for enableBot
- Local mode handler for disableBot
- Local mode handler for assignBot

#### Ticket Link

- https://mattermost.atlassian.net/browse/MM-25083
- https://mattermost.atlassian.net/browse/MM-25084
- https://mattermost.atlassian.net/browse/MM-25085
- https://mattermost.atlassian.net/browse/MM-25086
- https://mattermost.atlassian.net/browse/MM-25087

### CODE REVIEW

File: api4/bot_local.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+package api4
+
+func (api *API) InitBotLocal() {
+	api.BaseRoutes.Bot.Handle("", api.ApiLocal(patchBot)).Methods("PUT")
+	api.BaseRoutes.Bot.Handle("/disable", api.ApiLocal(disableBot)).Methods("POST")
+	api.BaseRoutes.Bot.Handle("/enable", api.ApiLocal(enableBot)).Methods("POST")
+	api.BaseRoutes.Bot.Handle("/assign/{user_id:[A-Za-z0-9]+}", api.ApiLocal(assignBot)).Methods("POST")
+
+	api.BaseRoutes.Bots.Handle("", api.ApiLocal(getBots)).Methods("GET")
```
Comment (isacikgoz) [inline api4/bot_local.go:12]: Should we also include `api.BaseRoutes.Bot.Handle("", api.ApiLocal(getBot)).Methods("GET")`?

Comment (ashishbhate) [reply api4/bot_local.go:12]: `getBot` gets a bot by it's userID. `getBots` gets by ownerId (with other search options). I couldn't find a Jira ticket for `getBot`, so I assumed it wasn't high priority for this epic.

Comment (isacikgoz) [reply api4/bot_local.go:12]: I see, we can add this in another PR.

**Pull request merged**